spec/config/locales/l10n/bundles/test/4.8.1/JAVA/messages_de.json

```diff
@@ -2,7 +2,8 @@
   "component" : "JAVA",
```

Comment:
Change `JAVA` to something else? This is a `Ruby` project, looks weird to have `JAVA` here.

Reply:
i think it's unnecessary, because it's just a name of component, it can be anything, and it's not related to code logic.